posthog/settings/web.py

```diff
@@ -400,6 +398,12 @@
 DEV_DISABLE_NAVIGATION_HOOKS = get_from_env("DEV_DISABLE_NAVIGATION_HOOKS", False, type_cast=bool)
 
 
+REMOTE_CONFIG_DECIDE_ROLLOUT_PERCENTAGE = get_from_env("REMOTE_CONFIG_DECIDE_ROLLOUT_PERCENTAGE", 0.0, type_cast=float)
+
+if REMOTE_CONFIG_DECIDE_ROLLOUT_PERCENTAGE > 1:
+    raise ValueError(
+        f"REMOTE_CONFIG_DECIDE_ROLLOUT_PERCENTAGE must be between 0 and 1 but got {REMOTE_CONFIG_DECIDE_ROLLOUT_PERCENTAGE}"
```

Comment:
Technically someone could make it `-1` and the code wouldn't complain

Reply:
yeah not worried about that. More worried that I forget how the setting works and use "10" instead of "0.1"